libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -25,20 +26,47 @@ const handleError = (error: any) => {
 
 @Injectable()
 export class UserProfileService {
-  constructor(private userProfileApi: UserProfileApi) {}
+  constructor(
+    private userProfileApi: UserProfileApi,
+    private readonly islyklarApi: IslyklarApi,
+  ) {}
 
   userProfileApiWithAuth(auth: Auth) {
     return this.userProfileApi.withMiddleware(new AuthMiddleware(auth))
   }
 
+  async getIslyklarData(ssn: User['nationalId']) {
+    try {
+      const islyklarData = await this.islyklarApi.islyklarGet({
+        ssn,
+      })
+
+      return {
+        mobilePhoneNumber: islyklarData?.mobile,
+        email: islyklarData?.email,
+      }
```

Comment:
We should also return `emailVerified: true` and `mobilePhoneVerified: true` if this returns data.

Reply:
Oh of course. Adding that in.